.github/workflows/ms.network.applicationgateways.yml

```diff
@@ -4,21 +4,21 @@ on:
   workflow_dispatch:
```

Comment:
Would you suggest to use double quotes instead of single? If that's due to automatic formatting I'd suggest to goback to single quotes for consistency with other workflows.

Reply:
Keeping as it was. That was a formatonsave configuration in the VSCode :)